tutorials/shading/shading_reference/spatial_shader.rst

```diff
@@ -272,7 +272,17 @@ To write a light function, assign something to DIFFUSE_LIGHT or SPECULAR_LIGHT.
 means no light is processed.
 
 The light function is called for every light in every pixel. It is called within a loop for
-each light type. 
+each light type.
+
+Light shader example:
+
+.. code-block:: glsl
+
+    void light() {
+        DIFFUSE_LIGHT += vec3( dot(NORMAL,LIGHT) * ALBEDO * ATTENUATION );
+    }
+
+Keep in mind DIFFUSE_LIGHT must be added to rather than set, if you don't want one light to overwrite the work of others.
```

Comment:
"DIFFUSE_LIGHT" should have two back ticks around it ``DIFFUSE_LIGHT``. 

Also, you should rephrase this to be less commanding and more positive. Maybe something like "If you want the lights to add together, add the light contribution to ``DIFFUSE_LIGHT`` using ``+=``, rather than overwriting it."